account_invoice_refund_option/models/account_invoice.py

```diff
@@ -0,0 +1,23 @@
+from odoo import Command, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.move"
+
+    def copy_data(self, default=None):
```

Comment:
why not inherit [def _prepare_default_reversal](https://github.com/odoo/odoo/blob/18.0/addons/account/wizard/account_move_reversal.py#L90) instead